security/cert_manager_operator/cert-manager-operator-release-notes.adoc

```diff
@@ -76,6 +76,31 @@ This is the general availability (GA) release of the {cert-manager-operator}.
 
 * After uninstalling the {cert-manager-operator} by using the web console, the {cert-manager-operator} does not remove the cert-manager controller, CA injector, and Webhook pods automatically from the `cert-manager` namespace. As a workaround, you can manually delete the cert-manager controller, CA injector, and Webhook pod deployments present in the `cert-manager` namespace. (link:https://issues.redhat.com/browse/OCPBUGS-13679[*OCPBUGS-13679*])
 
+[id="cert-manager-operator-release-notes-1.10.3"]
+== Release notes for {cert-manager-operator} 1.10.3
+
+Issued: 2023-08-08
+
+The following advisory is available for the {cert-manager-operator} 1.10.3:
+
+* link:https://access.redhat.com/errata/RHSA-2023:4335[RHSA-2023:4335]
+
+For more information, see the link:https://cert-manager.io/docs/release-notes/release-notes-1.10/#v1102-changes-since-v1101[cert-manager project release notes for v1.10.2]. The `1.10.3` version of the {cert-manager-operator} is based on the cert-manager upstream version `1.10.2`.
```

Comment:
```
The `1.10.3` version of the {cert-manager-operator} is based on the cert-manager upstream version `1.10.2`.
```
Though this is true, this could confuse readers. Need add a bit to explain it (ref: Swarup's https://redhat-internal.slack.com/archives/C045YMPKR3M/p1690780775293429?thread_ts=1690769314.085919&cid=C045YMPKR3M ; Thejas's https://github.com/openshift/cert-manager-operator/pull/133#issue-1785980725 )

Reply:
Then from the logically reading perspective, for 1.10.3 RN, swap the order of the first sentence "For more information, see ... v1.10.2" and the second sentence "... 1.10.3 ... is based on ... 1.10.2".